graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/java/TypeUtils.kt

```diff
@@ -41,7 +40,6 @@ class TypeUtils(private val packageName: String, private val config: CodeGenConf
         "TimeZone" to ClassName.get(String::class.java),
         "Date" to ClassName.get(LocalDate::class.java),
         "DateTime" to ClassName.get(OffsetDateTime::class.java),
-        "Currency" to ClassName.get(Currency::class.java),
```

Comment:
Actually, instead of removing this mapping, I think a better approach is to check if there is an existing type defined in the schema document and favor that, i.e. skip the check if it exists in the document. You can add that check something like this:
```
 !document.definitions.filterIsInstance<ObjectTypeDefinition>().any { e -> e.name == name } 
```
around this line: https://github.com/Netflix/dgs-codegen/blob/master/graphql-dgs-codegen-core/src/main/kotlin/com/netflix/graphql/dgs/codegen/generators/kotlin/KotlinTypeUtils.kt#L137

This way we still have fallback for the commonly used types and won't break existing builds. It will also handle the scenario for all the defined mappings.

Reply:
Yes, this makes way more sense. 
Taking into account that in my example is an Enum, but that can be a type or even a scalar (I think, I'm not that familiar with graphQL) I'm checking for all those data types. I've also added tests to check this works.
Let me know if this is ok :)
Thanks